std/parallelism.d

```diff
@@ -2864,13 +2864,13 @@ public:
             }
         }
 
-        ref T opIndex(size_t index)
+        ref T opIndex(size_t index) const
```

Comment:
But isn't `const` supposed to be transitive? How are you getting a non-const T out of a const function?